conf/report/report-template.html

```diff
@@ -20,6 +20,10 @@
   </head>
 
   <script>
+    if (process.env.BUILD_ID !== 'local') 
```

Comment:
This should operate on jinja2 template arguments, this will attempt to get the env of the user browser, we need to read those values from the environment of the build machine